content/_announcements/jan2020-new-website.md

```diff
@@ -4,25 +4,37 @@ title: Our new website
 
-Street art distillery migas schlitz tumeric hashtag pinterest food truck hammock sustainable chicharrones +1 slow-carb cardigan. Semiotics adaptogen austin slow-carb 3 wolf moon church-key tote bag, kale chips vegan chicharrones twee kogi tumblr iPhone fingerstache. 3 wolf moon photo booth meggings iceland succulents gastropub meditation swag hashtag mixtape helvetica aesthetic pabst. Lo-fi artisan PBR&B +1 kale chips. Scenester VHS knausgaard vaporware pop-up irony tofu schlitz. Keytar hot chicken poutine vexillologist glossier truffaut jianbing PBR&B lo-fi pork belly neutra echo park try-hard green juice.
+One reason for the site is to duplicate some of the features we've been using on Meetup.com, a subsidiary of the struggling startup WeWork.
 
-## Heading
+Lundeby has built a site with this unusual toolkit before. In 2018, after evaluating "a number of tools," including static site generators and CCMS systems, she decided to use Jekyll to build a site for the documentation she maintains for Veeva Systems, a Pleasanton-based cloud-computing company that builds applications for the pharmaceutical and life sciences industry.
 
-Portland hexagon put a bird on it green juice. Yr marfa twee man bun. Hexagon YOLO stumptown, succulents yr chia man bun. Irony knausgaard thundercats jean shorts ethical keffiyeh hashtag single-origin coffee cardigan deep v snackwave meditation poke XOXO pug. Chambray cardigan man braid crucifix flannel affogato keffiyeh schlitz.
+She chose Jekyll because it had a largest number of users and the largest support presence of all the likely options. That doesn't mean the job was easy.
 
-Tumeric leggings mlkshk tote bag. Poutine single-origin coffee authentic, disrupt humblebrag franzen XOXO viral raclette prism cardigan +1 polaroid. Viral deep v direct trade vape taxidermy vinyl readymade food truck, before they sold out ethical +1 shoreditch. Mlkshk stumptown kinfolk vice man bun helvetica street art af twee 3 wolf moon chambray irony. Affogato shoreditch truffaut plaid, man bun gentrify hoodie tousled sustainable neutra keffiyeh kitsch swag. Mixtape polaroid woke art party austin jianbing. Woke before they sold out gentrify vinyl you probably haven't heard of them fam lo-fi disrupt gochujang cold-pressed.
+With just a little previous exposure to HTML and CSS, Lundeby waded into Jekyll, only to start over again four or five times, after realizing that each subsequent theme she chose was not going to do the job the way she wanted. She eventually decided to build her own custom theme, which is what her Veeva docs site uses today.
 
-Oh. You need a little dummy text for your mockup? How quaint.
+Lundeby decided to use Bootstrap because the framework provides components like navbars and tables of contents without the need to write all the code for them. It also powers the reusable venue cards on our events pages, and the embedded Google maps.
```

Comment:
I'm not certain of the best way to state this, or even if it's needed, but the reusable nature is from Jekyll?

Reply:
Does that mean that Bootstrap provides some of the functionality and Jekyll makes it reusable, or is that feature all Jekyll?

Reply:
If that feature comes from Jekyll, can you tell me about some more UI features that rely on Bootstrap?

Reply:
Bootstrap is just the UI design only - visual and things like mobile responsiveness, collapse/expand, etc. Everything to do with the content comes from Jekyll.

The UI components we're using from Bootstrap currently are:
* Grid framework
* Navbar
* Cards
* Breadcrumb (style only, not the creation of the breadcrumb)
* Buttons
* and more.....

It's probably not really that productive to list out the UI components we're using from Bootstrap...maybe something like this instead:

Lundeby decided to use Bootstrap because the framework provides useful, pre-made components. Instead of building components such as the navbar from scratch, we were able to use Bootstrap components as a jumping off point instead.  (Learn more about [Bootstrap](https://getbootstrap.com/)!)